numpy/_core/src/multiarray/arraywrap.c

```diff
@@ -34,7 +33,7 @@ npy_find_array_wrap(
     PyObject *wrap = NULL;
     PyObject *wrap_type = NULL;
 
-    double priority = -DBL_MAX;  /* silence uninitialized warning */
+    double priority = -NPY_INFINITY;  /* silence uninitialized warning */
```

Comment:
Hmm, actually the comment should go - now one really would have a problem if the variable were uninitialized!